lib/reporting/cloudwatch_client.rb

```diff
@@ -165,9 +165,9 @@ def fetch_one(query:, start_time:, end_time:)
       wait_for_query_result(query_id)
     rescue Aws::CloudWatchLogs::Errors::InvalidParameterException => err
       if err.message.match?(/End time should not be before the service was generally available/)
-        # rubocop:disable Layout/LineLength
-        log(:warn, "query end_time=#{end_time} (#{Time.zone.at(end_time)}) is before Cloudwatch Insights availability, skipping")
-        # rubocop:enable Layout/LineLength
+        # rubocop:disable Layout/LineLength, Rails/TimeZone
+        log(:warn, "query end_time=#{end_time} (#{Time.at(end_time)}) is before Cloudwatch Insights availability, skipping")
+        # rubocop:enable Layout/LineLength, Rails/TimeZone
```

Comment:
Thanks, Rubocop, for complaining that we're not reintroducing the bug. 😆 

![gif of Robery Downey Jr. sighing and facepalming](https://media.giphy.com/media/116a8zosxwA0SI/giphy.gif)